### PR TITLE
Install OSSF Scorecard

### PR DESCRIPTION
**Pull request by ALRubinger**
Enables License and other checks

### CODE REVIEW

**Comment by mistermoe** [comment]
yo @ALRubinger , thanks for adding this! 

qq regarding:

image


I imagine this will fix itself when we merge to main?

File: .github/workflows/scorecard.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+jobs:
+  analysis:
+    name: Scorecard analysis
+    runs-on: ubuntu-latest
+    permissions:
+      # Needed to upload the results to code-scanning dashboard.
+      security-events: write
+      # Needed to publish results and get a badge (see publish_results below).
+      id-token: write
+      # Uncomment the permissions below if installing in a private repository.
```
Comment (KendallWeihe) [inline .github/workflows/scorecard.yml:29]: We can safely assume this'll always be public so we can delete these commented lines

**Review by KendallWeihe — APPROVED** [review]

**Review by michaelneale — APPROVED** [review]
lets do this soon

**Comment by ALRubinger** [comment]
> yo @ALRubinger , thanks for adding this!
> 
> qq regarding:
> 
> image
> I imagine this will fix itself when we merge to main?

Yup, should. Merging to main will trigger the checks and publish them - right now the badge can't see prior published results.

File: .github/workflows/scorecard.yml (hunk 1 above)
Comment (ALRubinger) [reply .github/workflows/scorecard.yml:29]: Let's leave 'em for documentation purposes.

**Comment by mistermoe** [comment]
@ALRubinger merge away

**Pull request merged**